docs/documentation.rst

```diff
@@ -7,27 +7,16 @@ Writing documentation
 Documentation of Plone
 ======================
 
-As a community,
-Plone maintains several types of documentation:
+The comprehensive ressource for Plone documentation is http://docs.plone.org/. The following sections among 
```

Comment:
Minor typos:

ressource -> resource
"documentation is http" -> *documentation is on http"